launcher/src/backend/ethereum-services/SSVNetworkService.js

```diff
@@ -30,20 +30,20 @@ MetricsAPIPort: 15000`
   }
 
   static buildByUserInput (network, ports, dir, executionClients, consensusClients) {
-    const service = new BloxSSVService()
+    const service = new SSVNetworkService()
     service.setId()
     const workingDir = service.buildWorkingDir(dir)
 
-    const image = 'bloxstaking/ssv-node'
+    const image = 'bloxstaking/ssv-node-shifu'
 
 
     const volumes = [
-      new ServiceVolume(workingDir + '/data/blox/ssv', '/data')
+      new ServiceVolume(workingDir + '/data/ssv/network', '/data')
```

Comment:
No need for `ssv/network` in the sub-directories. Just `workingDir + '/data'` is enough, don't you think?